Lib/enum.py

```diff
@@ -309,6 +310,11 @@ def __call__(cls, value, names=None, *, module=None, qualname=None, type=None, s
         return cls._create_(value, names, module=module, qualname=qualname, type=type, start=start)
 
     def __contains__(cls, member):
+        if not isinstance(member, Enum):
+            warnings.warn(
+                    "using non-Enums in containment checks will raise "
+                    "TypeError in 3.8+",
```

Comment:
"in Python 3.8"

Reply:
Done

Reply:
"in 3.8" is a jargon. Non-core developers could not understand it. It is better to be more explicit in public warnings: "in Python 3.8".

Reply:
Whoops, missed the "Python" part -- done now.